zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -117,97 +183,181 @@ def wrap(*args, **kwargs):
+        return jsonobject.dumps(rsp)
 
     @replyerror
     def ping(self, req):
         cmd = jsonobject.loads(req[http.REQUEST_BODY])
-        rsp = AgentResponse()
+        rsp = PingResponse()
+
+        current_cluster_uuid = zbsutils.get_cluster_uuid(cmd.clusterInfo.version)
+        if current_cluster_uuid != cmd.clusterInfo.uuid:
+            raise Exception('cluster uuid does not match, current cluster uuid[%s], old cluster uuid[%s]' % (
+            current_cluster_uuid, cmd.clusterInfo.uuid))
```

Ruff (0.14.14)

[warning] Create your own exception

(TRY002)